code/modules/surgery/helpers.dm

```diff
@@ -118,7 +118,13 @@
 
 
 /proc/get_pain_modifier(mob/living/carbon/human/M) //returns modfier to make surgery harder if patient is conscious and feels pain
-	if(M.stat) //stat=0 if CONSCIOUS, 1=UNCONSCIOUS and 2=DEAD. Operating on dead people is easy, too. Just sleeping won't work, though.
+	if(M.stat == DEAD) // Operating on dead people is easy
+		return 1
+	var/datum/status_effect/incapacitating/sleeping/S = M.has_status_effect(STATUS_EFFECT_SLEEPING)
+	if(M.stat == UNCONSCIOUS && !(S?.voluntary))
```

Comment:
```suggestion
	if(M.stat == UNCONSCIOUS && (!S || !S.voluntary))
```
This won't work. Doing a ternary on a boolean `FALSE` (`0`) will cause a runtime exception. Here you'll have to write it out. Or refactor the `has_status_effect` to return `null` instead.